specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-12-01/Marketplace.json

```diff
@@ -0,0 +1,1204 @@
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/{privateStoreId}/requestApprovals/{offerId}/query": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/OfferIdPathParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_GetOfferWithApprovalRequestsStatuses",
+        "description": "Get offer with approval request status foreach plan, this api is used as a complex GET operation.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreApprovalRequestProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreApprovalRequestResource"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreOfferWithRequestApproval": {
+            "$ref": "./examples/GetPrivateStoreOfferWithRequestApproval.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/{privateStoreId}/requestApprovals/{offerId}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/OfferIdPathParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "put": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_CreateApprovalRequest",
+        "description": "Create approval request",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreApprovalRequestProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreApprovalRequestResource"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "UpsertApprovalRequest": {
+            "$ref": "./examples/UpsertApprovalRequest.json"
+          },
+          "DeleteApprovalRequest": {
```

Comment:
Why is there a delete example in the PUT operation?  Deletes should use the DELETE API verb

Reply:
changed this to action - withdrawPlan